src/reducer.js

```diff
@@ -5,7 +5,7 @@ import { LOCATION_CHANGE } from './actions'
  * Utilises the search prop of location to construct query.
  */
 const injectQuery = (location) => {
-  if (!location) {
+  if (!location || typeof window === 'undefined' || typeof window.location === 'undefined') {
```

Comment:
You cannot return `location` here. You need to return an object like the following: `{ query: {} }`